lib/trino-filesystem-alluxio/pom.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.trino</groupId>
+        <artifactId>trino-root</artifactId>
+        <version>446-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>trino-filesystem-alluxio</artifactId>
+    <description>Trino Filesystem - Alluxio</description>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <air.compiler.fail-warnings>true</air.compiler.fail-warnings>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.google.inject</groupId>
+            <artifactId>guice</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-filesystem</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-memory-context</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.alluxio</groupId>
+            <artifactId>alluxio-core-client-fs</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.alluxio</groupId>
+            <artifactId>alluxio-core-common</artifactId>
```

Comment:
This uses `com.amazonaws:aws-java-sdk-core` which is the [deprecated V1 SDK](https://aws.amazon.com/blogs/developer/announcing-end-of-support-for-aws-sdk-for-java-v1-x-on-december-31-2025/). Is it needed? How is Alluxio using it? We're working to move everything to the V2 SDK.

Why are the following needed?

* `commons-cli:commons-cli`
* `io.prometheus:prometheus-metrics-core`
* `io.prometheus:prometheus-metrics-exporter-servlet-jakarta`
* `io.prometheus:prometheus-metrics-instrumentation-jvm`
* `io.swagger:swagger-annotations`
* `jakarta.servlet:jakarta.servlet-api`
* `org.reflections:reflections`

Reply:
ping

Reply:
aws excluded, all other unneeded dependencies excluded as well.